api/src/utils/media/csv/csv-file.ts

```diff
@@ -287,21 +287,29 @@ export class XLSXCSV implements IWorkbook {
   }
 }
 
-export type IHeaderErrorType = 'Invalid' | 'Missing';
-export type IHeaderErrorCode = 'DuplicateHeader' | 'UnknownHeader' | 'MissingRequiredHeader';
+export type IHeaderErrorCode =
+  | 'duplicate_header'
+  | 'unknown_header'
+  | 'missing_required_header'
+  | 'missing_recommended_header'
+  | 'miscellaneous_for_header';
+
+export type IRowErrorCode =
+  | 'missing_required_field'
+  | 'unexpected_formats'
+  | 'Values_exceeded'
```

Comment:
nit for consistency:
```suggestion
  | 'values_exceeded'
```